Patch-Source:
https://gitlab.gnome.org/World/Phosh/phosh/-/commit/26cea5baa6e4ad840c958f7ee3c48b0d0d5675d2,
https://gitlab.gnome.org/World/Phosh/phosh/-/commit/f8d0b2da186e97c98147a511f6f9bb85de2d223a
---
From 26cea5baa6e4ad840c958f7ee3c48b0d0d5675d2 Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Sun, 25 May 2025 22:15:28 +0200
Subject: [PATCH] data: Update list of adaptive apps

Loupe and Showtime already merged the patches to list their form
factors, while the MR in gnome-calendar is still open.

Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1723>
---
 data/00_mobi.Phosh.gschema.override | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/data/00_mobi.Phosh.gschema.override b/data/00_mobi.Phosh.gschema.override
index 8d54ba225..e33e4060f 100644
--- a/data/00_mobi.Phosh.gschema.override
+++ b/data/00_mobi.Phosh.gschema.override
@@ -29,4 +29,6 @@ power-button-action='nothing'
 unlock-sim=true
 
 [sm.puri.phosh:Phosh]
-force-adaptive=['firefox-esr.desktop', 'org.gnome.Calendar.desktop']
+# gnome-calendar: https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1697
+# Loupe and Showtime will be fixed with GNOME 49
+force-adaptive=['firefox-esr.desktop', 'org.gnome.Calendar.desktop', 'org.gnome.Loupe', 'org.gnome.Showtime']
-- 
GitLab

From f8d0b2da186e97c98147a511f6f9bb85de2d223a Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Mon, 26 May 2025 01:28:17 +0200
Subject: [PATCH] data: Fix typo in override gschema

Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1724>
---
 data/00_mobi.Phosh.gschema.override | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/00_mobi.Phosh.gschema.override b/data/00_mobi.Phosh.gschema.override
index e33e4060f..b34d026d3 100644
--- a/data/00_mobi.Phosh.gschema.override
+++ b/data/00_mobi.Phosh.gschema.override
@@ -31,4 +31,4 @@ unlock-sim=true
 [sm.puri.phosh:Phosh]
 # gnome-calendar: https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1697
 # Loupe and Showtime will be fixed with GNOME 49
-force-adaptive=['firefox-esr.desktop', 'org.gnome.Calendar.desktop', 'org.gnome.Loupe', 'org.gnome.Showtime']
+force-adaptive=['firefox-esr.desktop', 'org.gnome.Calendar.desktop', 'org.gnome.Loupe.desktop', 'org.gnome.Showtime.desktop']
-- 
GitLab

